FrameworkSystem/private/standardLogging/Backend/RemoteBackend.py

```diff
@@ -0,0 +1,42 @@
+"""
+ServerBackend wrapper
```

Comment:
I added a comment on this point. 
RemoteBackend is the link between the handler and the formatter
RemoteHandler is a custom handler to send log messages to a remote service of DIRAC
Is it clear ?